--- etc/ttys	2011-02-16 20:19:39.000000000 -0600
+++ /etc/ttys	2011-04-12 14:51:40.000000000 -0500
@@ -28,26 +28,26 @@
 #
 # If console is marked "insecure", then init will ask for the root password
 # when going to single-user mode.
-console	none				unknown	off secure
+console	none				unknown	off
 #
-ttyv0	"/usr/libexec/getty Pc"		cons25	on  secure
+ttyv0	"/usr/libexec/getty Pc"		cons25	on 
 # Virtual terminals
-ttyv1	"/usr/libexec/getty Pc"		cons25	on  secure
-ttyv2	"/usr/libexec/getty Pc"		cons25	on  secure
-ttyv3	"/usr/libexec/getty Pc"		cons25	on  secure
-ttyv4	"/usr/libexec/getty Pc"		cons25	on  secure
-ttyv5	"/usr/libexec/getty Pc"		cons25	on  secure
-ttyv6	"/usr/libexec/getty Pc"		cons25	on  secure
-ttyv7	"/usr/libexec/getty Pc"		cons25	on  secure
-ttyv8	"/usr/local/bin/xdm -nodaemon"	xterm	off secure
+ttyv1	"/usr/libexec/getty Pc"		cons25	on 
+ttyv2	"/usr/libexec/getty Pc"		cons25	on 
+ttyv3	"/usr/libexec/getty Pc"		cons25	on 
+ttyv4	"/usr/libexec/getty Pc"		cons25	on 
+ttyv5	"/usr/libexec/getty Pc"		cons25	on 
+ttyv6	"/usr/libexec/getty Pc"		cons25	on 
+ttyv7	"/usr/libexec/getty Pc"		cons25	on 
+ttyv8	"/usr/local/bin/kdm -nodaemon"	xterm	on
 # Serial terminals
 # The 'dialup' keyword identifies dialin lines to login, fingerd etc.
-ttyu0	"/usr/libexec/getty std.9600"	dialup	off secure
-ttyu1	"/usr/libexec/getty std.9600"	dialup	off secure
-ttyu2	"/usr/libexec/getty std.9600"	dialup	off secure
-ttyu3	"/usr/libexec/getty std.9600"	dialup	off secure
+ttyu0	"/usr/libexec/getty std.9600"	dialup	off
+ttyu1	"/usr/libexec/getty std.9600"	dialup	off
+ttyu2	"/usr/libexec/getty std.9600"	dialup	off
+ttyu3	"/usr/libexec/getty std.9600"	dialup	off
 # Dumb console
-dcons	"/usr/libexec/getty std.9600"	vt100	off secure
+dcons	"/usr/libexec/getty std.9600"	vt100	off
 # Pseudo terminals
 ttyp0	none			network
 ttyp1	none			network
