--- ./etc/rc.conf	2010-11-07 08:26:59.000000000 -0600
+++ /etc/rc.conf	2011-06-16 14:06:59.000000000 -0500
@@ -15,7 +15,30 @@
 
 # If this is not set to YES, the system will drop into single-user mode.
 #
-rc_configured=NO
+rc_configured=YES
 
 # Add local overrides below
 #
+apmd=YES
+cgd=YES
+critical_filesystems_local="$critical_filesystems_local  /usr /var"
+dhclient=YES
+dhclient_flags="-nw ath0"
+estd=YES
+hostname=localhost.localdomain
+ifconfig_ath0=up
+ifconfig_ipw0=down
+ifconfig_wm0=down
+lkm=YES
+mixerctl=YES
+ntpdate=YES
+pf=YES
+pflogd=YES
+pf_rules=/usr/local/etc/pf.conf
+powerd=YES
+sshd=YES
+swapoff=YES
+wpa_supplicant=YES
+wpa_supplicant_flags="-B -i ath0 -c /usr/local/etc/wpa_supplicant.conf"
+wscons=YES
+xdm=YES
